etc/config/c.amazon.properties

```diff
@@ -361,6 +361,15 @@ compiler.cmips564el.exe=/opt/compiler-explorer/mips64el/gcc-5.4.0/mips64el-unkno
 compiler.cmips564el.name=MIPS64 gcc 5.4 (el)
 compiler.cmips564el.semver=5.4
 
+###############################
+# ARM HPC
+group.armhpc.compilers=armhpc-ac193a64
```

Comment:
&armhpc needs to be added to group.ccross.compilers line 202

Reply:
Thx - done.